index.html

```diff
@@ -0,0 +1,380 @@
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title> My Portfolio </title>
+	</head>
+
+	<body>
+
+		<header id="header">
+			<nav id="navigation">
+				<ul>
+					<li><a href="index.html" id="logo"> My Logo</a></li>
+					<li><a href="" class="menu-link"><img src="./images/iconMenu.svg" alt="menu-icon"
+								class="menu-icon"></a></li>
+				</ul>
+				<div class="item-nav">
+					<ul>
+						<li class="nav-list"><a href="#" class="nav-link">Portfolio</a></li>
+						<li class="nav-list"><a href="#" class="nav-link">About </a></li>
+						<li class="nav-list"><a href="#" class="nav-link">contact</a></li>
+					</ul>
+				</div>
+			</nav>
+		</header>
+		<section class="presentation">
+			<div class="greeting">
+				<h1>I’m Mariane <br> Glad to see you!</h1>
+			</div>
+			<div class="self-description">
+				<p>
+					I’m a software developer! I can help you build a product , feature or website Look through some of
+					my work and
+					experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+					me.
+				</p>
+			</div>
+			<a class="btn-mediaConnect" href="#">LET’S CONNECT</a>
+			<ul class="media-list">
+				<li>
+					<a href="" class="media-link"><img src="./images/twitterIcon.svg" alt="twitterIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/linkedIncon.svg" alt="linkdlnIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/mailIcon.svg" alt="mailIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/githubIcon.svg" alt="githubIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/inconito.svg" alt="inconitoIcon"
+							class="media-icon"></a>
+				</li>
+			</ul>
+		</section>
+
+		<!-- Project section-->
+
+		<section id="section-project">
+			<div class="details-card">
+				<div class="project-image1"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Tonic</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="Cinf">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="pl">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+				</div>
+			</div>
```

Comment:
- [ ] I noticed that the background color of the cards is different from what is in the Figma design. Kindly refer to the  Figma Design and apply the right color to the background :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/220819741-4e0260ea-1e56-4e2e-bd61-8512523bb30f.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/220819713-a30d1a59-b523-416e-929a-3eb5dff14c86.png)